docs/api/utilities/serialisation.md

```diff
@@ -1,5 +1,13 @@
 # Serialisation
 
+::: equinox.default_serialise_filter_spec
+
+---
+
+::: equinox.default_deserialise_filter_spec
+
+---
+
```

Comment:
Can these go at the bottom of this page, as the other material on this page is more important.